libs/core/src/lib/busy-indicator/busy-indicator.component.scss

```
@@ -1,21 +1,25 @@
 @import 'fundamental-styles/dist/busy-indicator';
 
-.fd-busy-indicator {
+* + .fd-busy-indicator,
+* + .fd-busy-indicator-extended__label {
```

```suggestion
#{$siblingBusyIndicator},
#{$siblingBusyIndicator} + .fd-busy-indicator-extended__label {
```
I think this is a bug. It does not do as previously did. to cover previous behavior you can save `* + .fd-busy-indicator` in some variable and then use it like this here